src/config.ts

```diff
@@ -75,6 +76,10 @@ export interface AutocompleteConfig<State> {
    * Google Analytics configuration. Set to `false` to disable.
    */
   googleAnalytics?: GoogleAnalyticsConfig | boolean
+  /**
+   * Decorate each search hit before rendering
+   */
+  hitDecorators?: HitDecorator[]
```

Comment:
I am a bit confused. My understanding is this parameter will be supplied from the template configuration. Currently, we have priceDecorator provided out of the box. So, is this config parameter is something that can be both custom and build-in (priceDecorator)

Reply:
Also, the price currency format is something that we apply automatically when configured in admin. But this approach expects the decorator to be configured for it to be applied.

Reply:
price decorator based usage would be something like this:

```
hitDecorators: [
  priceDecorator()
]
```